src/components/test-finalisation/d255/d255.ts

```diff
@@ -67,6 +67,7 @@ export class D255Component implements OnChanges {
 
     // set default to false unless eyesight test failed
     this.d255 = !!this.eyesightTestFailed;
+    this.d255Change.emit(this.d255);
```

Comment:
This implementation is different to DES3, just checking that that's correct?

Reply:
yes @andrewsetterfield changed the implementation slightly due to issues in ionicn 5 but i found the `emit()` was missing which was causing a bug